views/sis/index.js

```diff
@@ -5,20 +5,66 @@
  */
 
 import React from 'react'
-import {StyleSheet} from 'react-native'
+import {
+  StyleSheet,
+  Text,
+  View,
+  Navigator,
+} from 'react-native'
 
 import TabbedView from '../components/tabbed-view'
+import Button from 'react-native-button'
 import tabs from './tabs'
-
+import {loadLoginCredentials} from '../../lib/login'
+import * as c from '../components/colors'
 const styles = StyleSheet.create({
   container: {
     flex: 1,
   },
+  error: {
+    marginTop: 50,
+    marginLeft: 20,
+    marginRight: 20,
+  },
+  errorButton: {
+    backgroundColor: c.denim,
+    width: 200,
+    color: c.white,
+    alignSelf: 'center',
+    height: 30,
+    paddingTop: 3,
+    marginBottom: 10,
+    marginTop: 10,
+    borderRadius: 6,
+    overflow: 'hidden',
+  },
 })
 
-export default function SISView() {
-  return <TabbedView style={styles.container} tabs={tabs} />
+export default function SISView({navigator, route}: {navigator: typeof Navigator, route: Object}) {
+  // Check if the user has a valid username and password
+  let {username, password} = loadLoginCredentials()
```

Comment:
We might also want to add a `loaded` state var, so we can show the "loading" screen until it's loaded the credentials – shouldn't take long, but better to be prepared than to have someone really confused by the flashing of the "please log in" screen.